src/Parser/Paladin/Protection/Modules/Features/Checklist.js

```diff
@@ -50,6 +58,32 @@ class Checklist extends CoreChecklist {
       },
     }),
 
+    new Rule({
+      name: (
+        <React.Fragment>
+          Mitigate incoming damage with <SpellLink id={SPELLS.SHIELD_OF_THE_RIGHTEOUS.id} /> and <SpellLink id={SPELLS.CONSECRATION_CAST.id} />
+        </React.Fragment> 
+      ),
+      description: (
+        <React.Fragment>
+          Maintain <SpellLink id={SPELLS.CONSECRATION_CAST.id} /> to reduce all incoming damage by a flat amount and use it as a rotational filler if necessary.<br/>
+          Use <SpellLink id={SPELLS.SHIELD_OF_THE_RIGHTEOUS.id} /> to flat out your physical damage taken or weave them into your rotation when you're about to cap charges.
+        </React.Fragment>
+      ),
+      requirements: () => {
+        return [
+          new Requirement({
+            name: <React.Fragment><SpellLink id={SPELLS.SHIELD_OF_THE_RIGHTEOUS.id} /> efficiency</React.Fragment>,
+            check: () => this.shieldOfTheRighteous.suggestionThresholds,
+          }),
+          new Requirement({
+            name: <React.Fragment><SpellLink id={SPELLS.CONSECRATION_CAST.id} /> efficiency</React.Fragment>,
```

Comment:
I'd suggest calling this 'uptime' instead to not confuse it with how SotR is calculated